torch/_inductor/codegen/triton.py

```diff
@@ -3839,6 +3851,12 @@ def store_cache():
             # generating out of range indices for later calls.
             ms = do_bench(lambda: call(wrapped_jit_function.clone_args(*args)[0]))
 
+            # overhead of cloning args gives bias for fusing the kernel
+            # in the case of mutating/in-placeable second fusion
+            # TODO - would be better as a hook in triton do_bench that reset
+            # the input values between benchmarking
+            ms = ms - do_bench(lambda: wrapped_jit_function.clone_args(*args))
```

Comment:
The cloning overhead should be the same for different choices anyway? That should not affect comparison of kernel latencies

Reply:
But most likely those in-place args should also become inplace args for the fused 'template + epilogue'?

It's fine to add this to be more safe though

Reply:
In the case of mm + relu the inplacing is on the output of template

Reply:
ohh, that's a good point